app/grandchallenge/components/models.py

```diff
@@ -1876,6 +1884,13 @@ def can_execute(self):
             .exists()
         )
 
+    @property
+    def requires_gpu_type(self):
+        if self.requires_gpu:
+            return self.desired_gpu_type
+        else:
+            return ""
+
```

Comment:
Is the plan to also migrate the `ComponentImage` model to only have a `requires_gpu_type` CharField and deprecate the `requires_gpu` and `desired_gpu` fields?

Reply:
Precisely, makes GPU selection much more intuitive when we have more than one option.